CMP1903_A1_2324/Game.cs

```diff
@@ -15,23 +15,32 @@ internal class Game
          * rolls could be continous, and the totals and other statistics could be summarised for example.
          */
 
+        //needs to be accessed by testing class
+        public Die one, two, three;
+        public int sum;
+        public int roll1, roll2, roll3;
+
         //Methods
         //handles creating die objects, calculating sum and displaying to console
         public void PlayGame()
         {
             //Creates three die objects
-            Die one = new Die();
-            Die two = new Die();
-            Die three = new Die();
-
-            //old way of recording sum
-            //int sum = RollDie(one, 1) + RollDie(two, 2) + RollDie(three, 3);
+            one = new Die();
+            two = new Die();
+            three = new Die();
 
             //Tells user to press enter, done to make numbers random as time passes changing the random seed
             Console.WriteLine("Press enter to roll each dice!");
```

Comment:
Unsure if this is meant to be here? - You don't have a ReadKey() or ReadLine() after this before the dice outputs are rolled.

Perhaps moving the ReadLine() above the WriteLine() immediately after calculating the sum would be more desirable?